### ISSUE TITLE
Logo chart is out of sync with poll chart

### ISSUE THREAD

**Issue by adamchainz** [six50/sixfifty#6]
<img width="769" alt="screen shot 2017-05-07 at 11 12 23" src="https://cloud.githubusercontent.com/assets/857609/25779787/1f61c384-3316-11e7-8636-e05a7b3a0d84.png">

46.9% Con versus 47.9% Con, which one is it? Is there a lag on the logo chart or something? Can't find explanation anywhere...

**Comment by danpalmer** [comment]
Yeah they come from two different data sources right now. I'll be linking them up soon, thanks for the report.
